### DISCUSSION TITLE
Please tell me your favorite musician(including band).

### DISCUSSION THREAD

**Discussion by SInadumi** [NAIST-SE/AttractingContributors#437] [General]
Please tell me your favorite MUSICIAN(including band). I wanna know a whole new world.
あなたの好きなミュージシャン(バンドでも大丈夫です)を教えてください。
https://github.com/SInadumi/AttractingContributors

**Comment by aru1702** [comment]
An old and legendary band from British, The Beatles, do you know them?

See my repo: https://github.com/aru1702/AttractingContributors-TouhouFumo
Hop into my discussion: https://github.com/NAIST-SE/AttractingContributors/discussions/407
